matter/wifi/wf200/sl_wfx_task.c

```diff
@@ -124,7 +126,7 @@ static void wfx_bus_task(void *p_arg)
 void wfx_bus_start()
 {
   wfx_bus_task_handle =
-    xTaskCreateStatic(wfx_bus_task, "wfxbus", BUS_TASK_STACK_SIZE, NULL, 2, busStack, &busTaskStruct);
+    xTaskCreateStatic(wfx_bus_task, "wfxbus", BUS_TASK_STACK_SIZE, NULL, TASK_PRIORITY2, busStack, &busTaskStruct);
```

Comment:
same comment as junior of the task priority

Reply:
done